CHANGELOG.md

```diff
@@ -20,6 +20,22 @@ Visit The National Archives' [documentation on OGL and Crown copyright](https://
 
 This was added in [pull request #2702: Allow localisation of content licence and copyright notices in Footer](https://github.com/alphagov/govuk-frontend/pull/2702).
 
+#### Use new override classes to apply static spacing
+
+You can now use static spacing override classes to apply spacing from [the static spacing scale](https://design-system.service.gov.uk/styles/spacing/#static-spacing) to elements of your design.
+
+The new classes start with: `govuk-!-static` followed by either `margin-` or `padding-`, and then a spacing unit number.
+
+To apply spacing in a single direction, include `left-`, `right-`, `top-`, or `bottom-` just before the spacing unit.
+
+For example:
+
+- `govuk-!-static-margin-9` will apply a 60px margin to all sides of the element at all screen sizes
+- `govuk-!-static-right-5` will apply 25px of padding to the right side of the element at all screen sizes
```

Comment:
```suggestion
- `govuk-!-static-padding-right-5` will apply 25px of padding to the right side of the element at all screen sizes
```

Reply:
Good catch. I'd spotted this and fixed in #2279 where I'd copied this from but forgot to make the same change here…